node-definitions/sonic/SONiCv.yaml

```diff
@@ -0,0 +1,61 @@
+id: SONiCv
+general:
+  nature: switch
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - Eth0
+      - Ethernet0
+      - Ethernet4
+      - Ethernet8
+      - Ethernet12
+      - Ethernet16
+      - Ethernet20
+      - Ethernet24
+      - Ethernet28
+    serial_ports: 2
+    default_count: 9
+    loopback:
+      - Loopback0
+ui:
+  visible: true
+  label_prefix: SONiCv-
+  icon: switch
+  label: SONiC Virtual Switch (VM)
+sim:
+  linux_native:
+    libvirt_domain_driver: kvm
+    driver: server
+    disk_driver: virtio
+    ram: 4096
+    cpus: 2
+    cpu_limit: 25
```

Comment:
We typically leave the CPU limit to 100 (it's in percent).  Why have it limited to 25%?

Reply:
Ha probably because I am running CML on a fanless 900 elite mini g9 with 96Gb of ram :)

Reply:
You can always override the CPU settings per instance.  Best to leave the default in the node def as 100.